scripts/config/webpack/config-server.js

```diff
@@ -224,6 +225,10 @@ module.exports = ({ minify } = {}) => {
                 allChunks: true,
                 disable: !isDev,
             }),
+            !isDev && gzip && new CompressionPlugin({
```

Comment:
We don't need this on the server because we won't be serving the server bundle through the network.